### PR TITLE
Various documentation fixes (#4956 #4997)

### PR DESCRIPTION
**Pull request by rglazer2**
Addresses Issues #4956 and #4997

### CODE REVIEW

File: doc/src/docs/InputOutputReference/01a-InputOutputReference.md [hunk 1]
```diff
@@ -649,7 +649,7 @@ The choice made for this field has important implications for modeling accuracy
 
 <table class="table table-striped">
 <tr>
-<th>Demanh Winhow</th>
+<th>Demand Winhow</th>
```
Comment (mjwitte) [inline doc/src/docs/InputOutputReference/01a-InputOutputReference.md:652]: @rglazer2 Hadn't noticed before, "Winhow" is also misspelled here --> Window

Comment (rglazer2) [inline doc/src/docs/InputOutputReference/01a-InputOutputReference.md:652]: @mjwitte Just made that fix.

File: doc/src/docs/InputOutputReference/01b-InputOutputReference.md [hunk 2]
```diff
@@ -7023,7 +7023,7 @@ Schedule:Compact,
 ```
 
 
-### DesignSpecification:ZoneAirDistribution
+#### DesignSpecification:ZoneAirDistribution
```
Comment (mjwitte) [inline doc/src/docs/InputOutputReference/01b-InputOutputReference.md:7026]: @rglazer2 Was this change intended?  I think three # is correct.

File: doc/src/docs/InputOutputReference/01d-InputOutputReference.md [hunk 3]
```diff
@@ -21687,19 +21687,19 @@ This is the maximum temperature of cooling water inlet or outlet that can occur
 
 #### Field: Electrical Efficiency Curve Name
 
-This is the name of Curve:Triquadratic object that defines the steady-state net electrical efficiency.   The electrical efficiency, <span>\({\eta_e}\)</span>, is a function of   the cooling water mass flow rate, <span>\({\dot m_{cw}}\)</span>, the temperature of the cooling water at the inlet, <span>\({T_{cw}}\)</span>, the steady-state net electrical power produced, <span>\({P_{net,ss}}\)</span>.
+This is the name of Curve:Triquadratic object that defines the steady-state net electrical efficiency.   The electrical efficiency, $\eta_e$, is a function of   the cooling water mass flow rate, $\dot m_{cw}$, the temperature of the cooling water at the inlet, $T_{cw}$, the steady-state net electrical power produced, $P_{net,ss}$.
 
-           <span>\({\eta_e} = f\left( {{P_{net,ss}},{{\dot m}_{cw}},{T_{cw}}} \right)\)</span>
+$${\eta_e} = f\left( {{P_{net,ss}},{{\dot m}_{cw}},{T_{cw}}} \right)$$
```
Comment (mjwitte) [inline doc/src/docs/InputOutputReference/01d-InputOutputReference.md:21693]: @rglazer2 In my md editor, some of the equations are rendering better now, but the subscripts etc. in the text are not yet.  There's a reason for using the span tags.  @Myoldmopar can you please advise here?  If you look at the v8.3.0 pdf I/O Ref section for Generator:MicroCHP, the equations and other equation-like snippets in the text are not rendering.  The changes in this vicinity are intended to correct that.

File: doc/src/docs/InputOutputReference/01c-InputOutputReference.md [hunk 4]
```diff
@@ -7428,6 +7428,10 @@ This field is the name of a fan (ref: Fan:OnOff, Fan:ConstantVolume, Fan: Variab
 
 This field allows the user to specify the coil operating options as one of the following options: **None**, **Heating**, **Cooling** or **HeatingAndCooling**. If **None** is selected, the unit ventilator does not have any coils, and any other input will be ignored. If either **Heating** or **Cooling** is selected, only a heating or cooling coil, respectively, is present. Thus, only four more inputs will be expected. If **HeatingAndCooling** is selected, both heating and cooling coil input must be entered, and the unit ventilator will have both a heating and a cooling coil.
 
+#### Field: Supply Air Fan Operating Mode Schedule Name
+
+This field is a schedule name (ref: Schedule) that determines whether the operating modes of unit heater’s On:Off fan. A schedule values of 0 indicates cycling fan operation (fan cycles with cooling/heating coil). A schedule values greater than 0 (usually 1 is used) indicate constant fan operation (fan runs continually regardless of coils operation). If this input field is left blank, then the operating mode of the OnOff fan type defaults to cycling operation.
+
```
Comment (mjwitte) [inline doc/src/docs/InputOutputReference/01c-InputOutputReference.md:7434]: @rglazer2 This has some grammer issues, e.g. "of unit heater's" --> "of the unit heater's", and "A schedule values of 0" --> "A schedule value of 0", and some more.  Please update this as needed and make similar changes back in unit ventilator where this came from.

**Comment by Myoldmopar** [comment]
@rglazer2 (@mjwitte, @JasonGlazer), we have another issue with CI that I am trying to resolve.  The test failures have nothing to do with your branch.  I'll let you know when they are resolved so we can get CI to re-run on your branch.

**Comment by Myoldmopar** [comment]
@rglazer2 This looks fine to go in.  Are you planning on adding anything to it?  I'm happy to review/merge if not.

**Comment by rglazer2** [comment]
@Myoldmopar I don't have anything else to add, however since I couldn't seem to render the equations correctly on any of my MD editors, its unclear if the first issue in #4997 has been solved.

File: doc/src/docs/InputOutputReference/01c-InputOutputReference.md [hunk 5]
```diff
@@ -7428,6 +7428,10 @@ This field is the name of a fan (ref: Fan:OnOff, Fan:ConstantVolume, Fan: Variab
 
 This field allows the user to specify the coil operating options as one of the following options: **None**, **Heating**, **Cooling** or **HeatingAndCooling**. If **None** is selected, the unit ventilator does not have any coils, and any other input will be ignored. If either **Heating** or **Cooling** is selected, only a heating or cooling coil, respectively, is present. Thus, only four more inputs will be expected. If **HeatingAndCooling** is selected, both heating and cooling coil input must be entered, and the unit ventilator will have both a heating and a cooling coil.
 
+#### Field: Supply Air Fan Operating Mode Schedule Name
+
+This field is a schedule name (ref: Schedule) that determines the operating modes of the unit heater’s On:Off fan. A schedule value of 0 indicates cycling fan operation (fan cycles with cooling/heating coil). A schedule value greater than 0 (usually 1 is used) indicates constant fan operation (fan runs continually regardless of the coils operation). If this input field is left blank, then the operating mode of the On:Off fan type defaults to cycling operation.
```
Comment (mjwitte) [inline doc/src/docs/InputOutputReference/01c-InputOutputReference.md:7433]: @rglazer2 Almost there - I think this one should say "unit ventilator's" instead of "unit heater's".

**Comment by mjwitte** [comment]
@Myoldmopar This is ready for your final review and merge, pending your check of the equation rendering in the Generator:MicroCHP section of 01d-InputOutputReference.md

**Comment by Myoldmopar** [comment]
@rglazer2 I created a PR into rglazer2 that fixes the equation rendering.  Unfortunately I am still working on general guidance to get equations to render properly.  Things are sketchy wrt that.  Anyway, if you merge [that PR](https://github.com/rglazer2/EnergyPlus/pull/1), it will pull those changes into your branch, which will then be reflected here in this PR, and I can finally merge it into NREL/EnergyPlus.

**Comment by Myoldmopar** [comment]
Thanks @rglazer2.  Since this _does_ touch the idd, I'll let the robots have a pass at it.  If it comes back clean, I'll merge it right in.
